packages/edit-post/src/components/layout/index.js

```diff
@@ -120,7 +113,24 @@ function useEditorStyles() {
 			} );
 		}
 
-		return hasThemeStyles ? editorSettings.styles : defaultEditorStyles;
+		const baseStyles = hasThemeStyles
+			? editorSettings.styles ?? []
+			: defaultEditorStyles;
+
+		// Add a constant padding for the typewritter effect. When typing at the
+		// bottom, there needs to be room to scroll up.
+		if (
+			! isZoomedOutView &&
+			! hasMetaBoxes &&
+			renderingMode === 'post-only' &&
+			! DESIGN_POST_TYPES.includes( postType )
+		) {
+			baseStyles.push( {
+				css: 'body{padding-bottom: 40vh}',
```

Comment:
This style was already applied in the custom "VisualEditor" component that existed in the post editor. But I removed that component (it was mostly useless).

That said, this particular style might need to move to the editor package at some point to apply to "posts" when they load in the site editor context as well.